packages/util-package-installer/README.md

```diff
@@ -0,0 +1,49 @@
+# Package Installer
+
+[![NPM version][badge-npm]][info-npm]
+[![Node version][badge-node]][info-node]
+![MIT License][badge-license]
+
+This package allows dynamic installation of other NPM packages.
+
+Internally it uses node's `child_process.spawn` because the recommended way of installing
+node modules dynamically is to use `npm` directly.
+
+It has one dependency and is intended to be as light as possible.
+
+It does not accept `npm` v1-style package names. If you don't know what they are,
+don't worry because they are old :)
+
+## Sample, simple usage (async/await style)
+
+The general idea is to pass it a `JSON.parse`d `package.json`;
+It will either return `stdout`, or throw an `Error`.
+
+```
+const install = require('@springernature/util-package-installer');
+const packageJSON = require('./path/to/package.json');
+(async () => {
+    let installResult;
+    try {
+        installResult = await install.peerDependencies(packageJSON);
```

Comment:
When I read this, I thought the only option was to `install.peerDependencies`. Maybe worth noting above this example which methods are actually exported - `install.dependencies`, `install.devDependencies` etc...